compiler/backend/presLangScript.sml

```diff
@@ -234,381 +234,341 @@ val exh_to_pres_exp_def = tDefine"exh_to_pres_exp"`
     (exh_to_pres_pat p, exh_to_pres_exp e)::exh_to_pres_pes pes)`
   cheat; 
 
-(* pres_to_json *)
-(* TODO: Add words *)
-val lit_to_value_def = Define`
-  (lit_to_value (IntLit i) = Int i)
-  /\
-  (lit_to_value (Char c) = String [c])
-  /\
-  (lit_to_value (StrLit s) = String s)
-  /\
-  (lit_to_value _ = String "word8/64")`;
+(* Helpers for converting pres to structured. *)
```

Comment:
When we created presLang, we eventually decided to put all `*_to_pres` functions in `presLangScript`. Should we not put all `*_to_structured` functions in `structuredLangScript`, then? To me, it feels maybe a little out of place to have them in `presLangScript`.